sadcowpng/sharedspaces/views.py

```diff
@@ -258,3 +258,23 @@ def update_space_date_time(request, data_time_id):
                    "id": data_time_id}
 
     return render(request, 'sharedspaces/update_space_date_time.html', context=context)
+
+
+@login_required
```

Comment:
You could also add a user_is_entry_owner decorator after login_required too.

Reply:
Fixed it in feature/space-reuse